internal/fixture/state.go

```diff
@@ -100,7 +100,7 @@ func (s *State) UpdateObject(tv *typed.TypedValue, version fieldpath.APIVersion,
 	return nil
 }
 
-// Update the current state with the passed in object
+// Set the current state with the passed in object
```

Comment:
You've updated the comment but it doesn't match the name of the function anymore?

Reply:
I don't think this should be renamed actually.